features/bootstrap/VirtualMetricHandleContext.php

```diff
@@ -20,7 +20,7 @@ class VirtualMetricHandleContext extends CentreonContext
     public function iAddAVirtualMetric()
     {
         $this->page = new MetricsConfigurationListingPage($this);
-        $this->assertFind('css', 'a[class="btc bt_success"]')->click();
+        $this->assertFind('css', 'a[class="btc bt_success ms-1"]')->click();
```

Comment:
I see your question from the daily meeting. In that case isn't it better to replace `class=` by `class*=` (see the suggestion) in order to make the assert more robust ?
```suggestion
        $this->assertFind('css', 'a[class*="btc bt_success"]')->click();
```

Reply:
You're right Joffrey, I correct !